elements/pfe-modal/src/pfe-modal.scss

```diff
@@ -139,5 +182,15 @@ $LOCAL-VARIABLES: (
 			height: pfe-local(size, $region: close);
 			width: 	pfe-local(size, $region: close);
 		}
+
+		:host([pfe-media]) & {
+			top: 0px; 
+			right: 0;
+			border-radius: 3px;
+			padding: 7px;
+			--pfe-modal__close--size: 15px;
+			--pfe-modal__close--Padding: 10px;
+			--pfe-modal__close--BackgroundColor: #ffffff63;
```

Comment:
including opacity in the hex is not supported in IE11 if you want to add a browser query fallback: https://caniuse.com/#search=hex%20opacity

Reply:
ack, thanks! replaced with rgba